packages/ui-components/src/components/HeaderContainer/index.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export { HeaderContainer, type HeaderContainerProps } from "./HeaderContainer.component"
```

Comment:
Do we need to export it with a type?

Reply:
I'm not sure about that. I have looked at the other containers in the AppShell and except for `MainContainerInner`, `type` is used everywhere. In my research, I have not found any reason that speaks against it. As an expert in this topic, can you please clarify?

Reply:
I was also wondering... now that you brought it up, @barsukov... I tried without, looks like we need to use the keyword. 🤷‍♀️
<img width="949" alt="Screenshot 2024-11-19 at 11 21 51" src="https://github.com/user-attachments/assets/cfedfdff-c829-415d-b38b-ee7101670326">

Reply:
But I guess this is re-exporting.

Reply:
Should we be doing this in all components we're migrating, @barsukov? Exporting the type from the component index file.

Reply:
I think in general this concept of index files are anoying in a way I will just prefer to do export types directly, but it is ok. It looks like it is a typescript setting. So leave the type